catalog/catalog.protocol.md

```diff
@@ -112,24 +112,24 @@ The catalog service may require an authorization token. Details for including th
 
 This section describes how the IDS Information Model maps to DCAT resources.
 
-### 3.1 Asset Entry
+### 3.1 Dataset
 
-An `Asset Entry` is a [DCAT Dataset](https://www.w3.org/TR/vocab-dcat-3/#Class:Dataset) with the following attributes:
+A `Dataset` is a [DCAT Dataset](https://www.w3.org/TR/vocab-dcat-3/#Class:Dataset) with the following attributes:
 
 #### 3.1.1 odrl:hasPolicy
 
-An asset entry Dataset must have 1..N `hasPolicy` attributes that contain an ODRL `Offer` defining the usage control policy associated with the asset. Offers must NOT contain any
-target attributes. The target of an offer is the asset associated with the containing asset entry.
+A `Dataset` must have 1..N `hasPolicy` attributes that contain an ODRL `Offer` defining the usage control policy associated with the dataset. Offers must NOT contain any
+target attributes. The target of an offer is the associated dataset.
 
-> Note: As `odrl:hasPolicy rdfs:domain odrl:Asset` and `AssetEntry isA dcat:Dataset`, each `Asset Entry` is also an `odrl:Asset` from an ODRL perspective.
+> Note: As `odrl:hasPolicy rdfs:domain odrl:Asset` and `dspace:Dataset isA dcat:Dataset`, each `Dataset` is also an `odrl:Asset` from an ODRL perspective.
```

Comment:
We don't define the class `dspace:Dataset` but only the property `dspace:dataset`, so I suggest removing the "and `dspace:Dataset isA dcat:Dataset`" part